ts/backend/src/app/features/controller/submission.controller.mts

```diff
@@ -23,6 +23,56 @@ export class SubmissionController extends Controller {
     this.attachPatch('/result', this.patchResult)
   }
 
+  /**
+   * @swagger
+   * /submissions:
+   *  post:
+   *    description: Inserts new submission.
+   *    security:
+   *      - oauth2: []
+   *    requestBody:
+   *      required: true
+   *      content:
+   *        application/json:
+   *          schema:
+   *            type: object
+   *            properties:
+   *              name:
+   *                type: string
+   *                description: Display name of submission.
+   *              benchmark:
+   *                type: number
+   *                description: ID of benchmark this submission belongs to.
+   *              submission_image:
+   *                type: string
+   *                description: URL of submission executable image.
+   *              code_repository:
+   *                type: string
+   *                description: URL of submission code repository.
+   *              tests:
+   *                type: array
+   *                items:
+   *                  type: number
+   *                description: IDs of tests to run.
+   *    responses:
+   *      200:
+   *        description: Created.
+   *        content:
+   *          application/json:
+   *            schema:
+   *              allOf:
+   *                - $ref: "#/components/schemas/ApiResponse"
+   *                - type: object
+   *                  properties:
+   *                    body:
+   *                      type: array
+   *                      items:
+   *                        type: object
+   *                        properties:
+   *                          uuid:
+   *                            type: string
+   *                            description: UUID of submission.
```

Comment:
https://stackoverflow.com/questions/50204588/how-to-define-uuid-property-in-json-schema-and-open-api-oas#50210623
```suggestion
   *                            type: string
   *                            format: uuid
   *                            description: UUID of submission.
```